### DISCUSSION TITLE
Fetch request to external API running twice

### DISCUSSION THREAD

**Discussion by Zlkva** [vercel/next.js#34496] [Help] [answered]
I'm having an issue when submitting a fetch request to my API (GET w/ Bearer Auth)

When I submit my request (through a button click), my external API is showing two GET requests are being made, one is authenticated with my bearer auth token and one isn't. 

I've attempted to do some debugging on my own but I've hit a bit of a rough patch. 

Things I've tried:
Changing my fetch library - I'm using Isomorphic Unfetch but have also tried Axios with the same results.
Checking chrome network tab - Chrome is showing only 1 attempt with a single pre-flight.
Accessing the API with a different method - I've used both Python and Postman to submit the same request and only see 1 single GET request. 

The code I'm using:

I'm running my fetch request via a button press
`onClick={(()=>{this.loadUserData(1,1,false)})}`

which in turn runs

    loadUserData = (limit, skip, reload) => {
    let cc = JSON.parse(Cookies.get(process.env.NEXT_PUBLIC_LOGGEDINCOOKIE))

    fetch(`${process.env.NEXT_PUBLIC_APILOCATION}/myapi?limit=${limit}&skip=${skip}`, {
      method: 'GET',
      headers: {
        'Authorization': `Bearer ${cc.token}`,
        'Content-Type': 'application/json',
        'Accept': 'application/json, application/xml'
      }
    })
    .then(response => response.json())
    .then((json) => {
      if (reload === 'true') {
        if (json.length === 0) {
          return
        } else {
          this.setState({
            log: json
          })
          return    
        }
      } else {
        this.setState({
          log: json
        })
        return
      }
    })
    .catch((err)=>{
      console.error('Error contacting server, contact administrator')
      return
    }); 
    }

In all it's a pretty simple request.

I'm left guessing this is a Next/React behavior. Has anyone else encountered a similar issue?

**Comment by Zlkva** [comment, marked as answer]
I ran a fresh install of Next and dropped my pages into it, suddenly the duplicate request is gone.
I'm slowly trying to triage the situation, but that seems to have resolved the issue. I'm somewhat baffled and unsure of what the issue could have been.

----

I ended up deleting my .next folder and the duplicate fetch requests stopped.

**Comment by DotenvAymane** [comment]
I still have this issue
I tried deleting the .next folder yet it still duplicates 
m making an api call with the getServerSide function
any news about this ?
Thank you
